project_scrum/project_scrum.py

```diff
@@ -0,0 +1,500 @@
+             (40, '40'),
+             (100, '100'),
+             (00, 'Not Set'),
+             ], string='Value')
+    risk = fields.Selection([
+            (0, '0'),
+            (1, '1'),
+            (2, '2'),
+            (3, '3'),
+            (5, '5'),
+            (8, '8'),
+            (13, '13'),
+            (20, '20'), 
+            (40, '40'),
+            (100, '100'),
+            (00, 'Not Set'),
+            ], string='Risk')
+    kano = fields.Selection([
+            ('excitement', 'Excitement'),
+            ('indifferent', 'Indifferent'),
+            ('mandatory', 'Mandatory'), 
+            ('performance', 'Performance'),
+            ('questionable', 'Questionable'),
+            ('reverse', 'Reverse'), 
+             ('not_set', 'Not Set'),
+            ], string='Kano')
+    color = fields.Integer(related='project_id.color')
+    reference = fields.Char('Number', select=True, readonly=True, copy=False, default=':')
+
+    @api.depends('sprint_id')
+    @api.one
+    def _current_sprint(self):
+        sprint = self.env['project.scrum.sprint'].get_current_sprint(self.project_id.id)
+        #~ _logger.error('Task computed %r' % self)
+        if sprint:
+            self.current_sprint = sprint.id == self.sprint_id.id
+        else:
+            self.current_sprint = False
+            
+    def _search_current_sprint(self, operator, value):
+        #~ raise Warning('operator %s value %s' % (operator, value))
+        project_id = self.env.context.get('default_project_id', None)
+        sprint = self.env['project.scrum.sprint'].get_current_sprint(project_id)
+        #~ raise Warning('sprint %s csprint %s context %s' % (self.sprint_id, sprint, self.env.context))
+        _logger.error('Task %r' % self)
+        return [('sprint_id', '=', sprint and sprint.id or None)]
+            
+    @api.multi
+    def write(self, vals):
+        if vals.get('stage_id') == self.env.ref('project.project_stage_data_2').id:
+            vals['date_end'] = fields.datetime.now()
+        return super(project_task, self).write(vals)
+
+    @api.model
+    def _read_group_sprint_id(self, present_ids, domain, **kwargs):
+        project = self.env['project.project'].browse(self._resolve_project_id_from_context())
+
+        if project.use_scrum:
+            sprints = self.env['project.scrum.sprint'].search([('project_id', '=', project.id)], order='sequence').name_get()
+            return sprints, None
+        else:
+            return [], None
+
+    @api.model
+    def _read_group_us_id(self, present_ids, domain, **kwargs):
+        project = self.env['project.project'].browse(self._resolve_project_id_from_context())
+
+        if project.use_scrum:
+            user_stories = self.env['project.scrum.us'].search([('project_id', '=', project.id)], order='sequence').name_get()
+            return user_stories, None
+        else:
+            return [], None
+
+class project_actors(models.Model):
+    _name = 'project.scrum.actors'
```

Comment:
Please follow Odoo conventions instead of exceptions.
Each record of this model is a record of project.scrum.actor kind/type